### ISSUE TITLE
JDBC java.sql.ResultSet wrapper

### ISSUE THREAD

**Issue by noahlz** [RedBeardLab/JRediSQL#2]
It would be really great if there was some kind of wrapper class that makes the data returned from redis appear to be `java.sql.ResultSet` - then this could be closer to a drop-in replacement for a traditional SQL database connection. 

So instead of calling get(1) get(2) as in the example, you would iterate over `ResultSet` objects, with functions delegating to the underlying redis interface. Basically RedisResultSetDecorator.

**Comment by siscia** [comment]
I will look into this! 

I love this kind of feedback since we are not really a Java shop so we lack the experience to make this kind of decisions

**Comment by siscia** [comment]
Hey @noahlz 

it seems a quite bit of work, maybe it is me not having yet clear how to implement it. It is me misunderstanding something, or it is actually not as straightforward?

Do you think you can provide a PR? Or can you guys support the development maybe buying a license?

Cheers,

**Comment by gkorland** [comment]
Yes the JDBC API is pretty broad, but it's almost a must for Java developers so it can play nicely with all the ecosystem, perhaps we can fork the sqllite jdbc client and adapt it.
